pkg/Microsoft.ML.TensorFlow.Redist/Microsoft.ML.TensorFlow.Redist.nupkgproj

```diff
@@ -4,7 +4,7 @@
     <Authors>The TensorFlow Authors</Authors>
     <TargetFramework>netstandard2.0</TargetFramework>
     <PackageDescription>$(MSBuildProjectName) contains the TensorFlow C library version $(TensorFlowVersion) redistributed as a NuGet package.</PackageDescription>
-    <PackageLicenseUrl>https://github.com/tensorflow/tensorflow/blob/master/LICENSE</PackageLicenseUrl>
+    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
```

Comment:
NOTE: a difference I see between

https://github.com/aspnet/AspNetCore/blob/master/LICENSE.txt
and
https://github.com/tensorflow/tensorflow/blob/master/LICENSE

is that the TensorFlow one has a Copyright at the top

```Copyright 2018 The TensorFlow Authors.  All rights reserved.```

We do download the actual TF license (see line 22 in this file). Maybe we should just be including that as the `PackageLicenseFile`, and not use this `PackageLicenseExpression`, since the license may actually be different.

NOTE: I think there's an issue on how we are downloading that license file. The way we currently have it, we are downloading the GitHub HTML page with the license information. But instead, we should be getting the content of the file.
Basically, we should change
https://github.com/dotnet/machinelearning/blob/1b6b3c3ab22a79e593913fbc9397cef93256aa9c/src/Redist/Microsoft.ML.TensorFlow.Redist/Microsoft.ML.TensorFlow.Redist.proj#L13

to use ```https://raw.githubusercontent.com/tensorflow/tensorflow/master/LICENSE``` instead.